src/const.ts

```diff
@@ -0,0 +1,2 @@
+export const appName = "Emails and messages";
+export const appBrandColor = "rgb(199, 159, 58)";
```

Comment:
I'd rather stop using convention like `const`, `util`, `lib` etc. Its not *that* important, but for new code I'd avoid it. Its not cohesive and it doesn't scale well

We can name it like `app-metadata` or something like this. I'd also export `const AppMetadata {name, brandColor}` instead single members, to reduce number of public exports

Reply:
Not a fan of metadata in the name, we already have metadata in the API. WDYT `app-branding`? 
The number of exports here doesnt seem to be massive, I kind of prefer having `appName` auto completed/imported by the editor instead of pulling it from the editor